tests/test_base_scanner.py

```diff
@@ -197,7 +195,9 @@ def test_included_paths_is_empty_if_not_specified(self):
     def test_include_paths_are_calculated_if_specified(
         self, mock_compile: mock.MagicMock
     ):
-        self.options.include_path_patterns = ("foo",)
+        self.options.include_path_patterns = tuple(
+            [{"path-pattern": "foo", "reason": "Testing exclude path pattern"}]
+        )
```

Comment:
This works but I had to read it 3 times and then try it myself to convince myself it did what was needed. How about:
```suggestion
        self.options.include_path_patterns = (
            {"path-pattern": "foo", "reason": "Testing exclude path pattern"},
        )
```